docs/editors/vscode.md

```diff
@@ -255,6 +305,19 @@ To assign shortcuts just go to the Keyboard Shortcuts page (`File` ->
 `Preferences` -> `Keyboard Shortcuts`) and search for a command, click on it and
 use your preferred shortcut.
 
+### Defining jvm options
+
+Unfortunately, it's not always possible to define environment variables or jvm
+options for tests. To work around that you can use:
+
+- `.jvmopts` file inside the main project directory or `JVM_OPTS` environment
+  variable. In this case, we will filter out any -X options as it might sometime
+  be problematic if the file is also used for specifying build tools options.
+- `.test-jvmopts` file or `TEST_JVM_OPTS` if you want to declare jvm options
+  only for your tests and you also want to use -X options.
```

Comment:
```suggestion
  only for your tests and/or you also want to use -X options.
```
Maybe make it more explicit that this will override `.jvmopts`.

Reply:
done